consul_config.py.ctmpl

```diff
@@ -39,6 +39,7 @@ TIMESCALE_ADMIN_URI="postgresql://postgres:postgres@{{.Address}}:{{.Port}}/postg
 TIMESCALE_ADMIN_LB_URI="postgresql://postgres:postgres@{{.Address}}:{{.Port}}/listenbrainz_ts"
 {{end}}
 {{end}}
+SQLALCHEMY_TIMESCALE_URI = ""
```

Comment:
If we move this line before the service block, we should be able to stop and start the containers without rebuilding images once the db is back up. Plus, if sometime the timescale db goes down we get a nice error page on the website itself rather than a hung up container.

Reply:
interesting idea - this ties in with my future plans about making LB able to start up if it's missing its services.

One unknown here is that I don't know when the service will disappear from consul (or when it'll come back up). I understand that it'll happen when the docker container starts up again. One nice thing about an explicit value here is that we can turn it on when we want to, rather than it depending on the startup of another service... 

I did actually originally think that the value in consul was an explicit key that was pasted directly into this string, but now I see that it's not as easy as that.